### PR TITLE
Add the option to display the Speed at the Checkpoints

### PR DESCRIPTION
**Pull request by Schmalzahar**
Since Nadeo removed the option to check the speed when you pass a checkpoint and the checkpoint speed is really helpfull when hunting a track, I added the option.

### CODE REVIEW

**Comment by Electron-x** [comment]
Thank you very much for the pull request. I will review it in the next few days.
A few small things have already come to my attention.
Especially the column constants should not be changed. COL_CPSPEED should get the not yet used value 0x8000, so that the user settings are correctly reloaded from the registry during an update installation.

File: DoTelemetry.cpp [hunk 1]
```diff
@@ -234,6 +234,15 @@ void DoTelemetry(STelemetryData* pTelemetry)
 			if (dwColumns & COL_CHECKPOINTS)
 				ListView_AddCheckpointTime(hwndListView, nRaceNumber, COLUMN_AUTOFIT, uCurrentNbCheckpoints,
 					pTelemetry->Current.Race.CheckpointTimes[uCurrentNbCheckpoints - 1]);
+			if (dwColumns & COL_CPSPEED) {
```
Comment (Electron-x) [inline DoTelemetry.cpp]: Please insert the empty line before the _if_ statement and use the Allman style for indentation (https://en.wikipedia.org/wiki/Indentation_style#Allman_style).

File: Helper.h [hunk 2]
```diff
@@ -45,6 +46,7 @@ const TCHAR szTrackName[] = TEXT("Map Name");
 const TCHAR szPlayerModel[] = TEXT("Player Model");
 const TCHAR szSector[] = TEXT("Sector %d");
 const TCHAR szCheckpoint[] = TEXT("CP %d");
+const TCHAR szCheckpointSpeed[] = TEXT("Speed %d");
```
Comment (Electron-x) [inline Helper.h]: If the display of checkpoints and sector times are switched off, a renaming to "Speed CP%d" might be better.

File: Helper.h [hunk 3]
```diff
@@ -29,6 +29,7 @@ const TCHAR szNbLaps[] = TEXT("Lap: %d/%d");
 const TCHAR szNbCheckpoints[] = TEXT("CP: %d");
 const TCHAR szSpeedKmh[] = TEXT("%d km/h");
 const TCHAR szSpeedMph[] = TEXT("%d mph");
+const TCHAR szSpeedmps[] = TEXT("%d m/s");
```
Comment (Electron-x) [inline Helper.h]: Please remove this unused string constant.

**Comment by Electron-x** [comment]
In addition to the individual review comments, please make the following changes:
Please remove the TMTelemetry.vcxproj file to remain backward compatible with VS2017.
In the file Resource.h, line 48, please change the tabs to spaces and change the value of the constant _APS_NEXT_COMMAND_VALUE in line 57 to 32798 (increment by 1).

**Comment by Schmalzahar** [comment]
> In addition to the individual review comments, please make the following changes:

I dont know if im missing something, but I dont see individual review comments.

**Comment by Electron-x** [comment]
I forgot to press a button. This is all new to me. I hope everything is visible now.
Sorry about the small details. But I'd also like to use your pull request to become more familiar with Github.

**Comment by Schmalzahar** [comment]
No problem, im new with github and with cpp. I have applied your suggestions for improvement. Once this is resolved, I have a new Pull Request comming up to highlight the current best CP speed, but this required more changes to the code.

**Pull request merged**